### ISSUE TITLE
`rasa data validate` returns 0 code even if error in data was found

### ISSUE THREAD

**Issue by IzidoroBaltazar** [RasaHQ/rasa#4543]
**Description of Problem**:

`rasa data validate` returns 0 code even if error was found.
This behavior is problematic when rasa is part of CI/CD pipeline and it is necessary to have consistent data. Current situation requires parsing of rasa output (custom script).

**Overview of the Solution**:

Handle `rasa data validate` so that it returns non 0 code when problem with data found.
I can implement solution to this problem if you decide to go with the change.

**Examples** (if relevant):

```
(latest_rasa) ubuntu@ip-172-31-38-153:~/sample-training-data$ rasa data validate --data training_data/ --domain configuration/domain.yml
2019-10-01 05:54:32 INFO     rasa.core.validator  - Validating intents...
2019-10-01 05:54:32 WARNING  rasa.core.validator  - The intent 'user.talk_human' is listed in the domain file, but is not found in the NLU training data.
2019-10-01 05:54:32 ERROR    rasa.core.validator  - The intent 'user.greet' is in the NLU training data, but is not listed in the domain.
2019-10-01 05:54:32 WARNING  rasa.core.validator  - The intent 'talk_human' is not used in any story.
2019-10-01 05:54:32 INFO     rasa.core.validator  - Validating utterances...
2019-10-01 05:54:32 WARNING  rasa.core.validator  - The utterance 'utter_agent.coffee' is not used in any story.
2019-10-01 05:54:32 WARNING  rasa.core.validator  - The utterance 'utter_default' is not used in any story.
(latest_rasa) ubuntu@ip-172-31-38-153:~/sample-training-data$ echo $?
0
(latest_rasa) ubuntu@ip-172-31-38-153:~/sample-training-data$
```

**Blockers** (if relevant):

**Definition of Done**:

**Comment by sara-tagger** [comment]
Thanks for submitting this feature request 🚀@wochinge will get back to you about it soon!✨

**Comment by wochinge** [comment]
@IzidoroBaltazar Sorry for my delayed answer! Which version of Rasa are you running? We implemented your suggested behavior in Rasa 1.3.0 :-)
